jobdsl/organisations-beta.groovy

```diff
@@ -172,15 +184,14 @@ Closure githubOrg(Map args = [:]) {
                 traits << 'org.jenkinsci.plugins.github__branch__source.ExcludeArchivedRepositoriesTrait' {
                 }
 
-                traits << 'io.jenkins.plugins.checks.github.status.GitHubSCMSourceStatusChecksTrait' {
-                    if (config.nightly) {
+                if (config.nightly) {
+                    traits << 'io.jenkins.plugins.checks.github.status.GitHubSCMSourceStatusChecksTrait' {
                         // TODO enable skip globally at some point so we don't have 2 job statuses
                         // not doing right now as tons of people will have it in their required commit statuses
                         skipNotifications(true)
                         def label = runningOnSandbox ? "Jenkins - sandbox nightly" : "Jenkins - nightly"
                         name(label)
                     }
-                    skipProgressUpdates(true)
```

Comment:
I don't think we need this anymore, the root cause of the issue was found and fixed